src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,44 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringClassTest {
+
+    @Test
+    @DisplayName("split() 메소드 테스트")
+    void testSplit() {
+        String testString1 = "1,2";
+        String testString2 = "1";
+
+        assertThat(testString1.split(",")).contains("1", "2");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?